code_samples/api/public_php_api/src/Command/ViewContentMetaDataCommand.php

```diff
@@ -86,7 +86,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln(' in ' . $versionInfo->getInitialLanguage()->name);
         }
 
-        $versionInfoArray = $this->contentService->loadVersions($contentInfo, VersionInfo::STATUS_ARCHIVED);
+        $versionInfoArray = iterator_to_array($this->contentService->loadVersions($contentInfo, VersionInfo::STATUS_ARCHIVED));
```

Comment:
Watch out - this code can break on PHP 8.2.

`iterator_to_array` can be used "safely" on arrays (which is one component of `iterable` (`Traversable|array`)) only on PHP 8.3.

That's why when performing PHPStan analysis we are often using multiple PHP versions.

Take care when backporting.

```suggestion
        $versionInfoArray = $this->contentService->loadVersions($contentInfo, VersionInfo::STATUS_ARCHIVED);
        if ($versionInfoArray instanceof \Traversable) {
            $versionInfoArray = iterator_to_array($versionInfoArray);
        }
```

Reply:
Thanks, I didn't know about it! I thought adjusting the samples for multiple versions might be too tiresome to begin with (especially regenerating the baseline locally for multiple versions is not pleseant I think), but I'll think about introducing it in the future